unstructured/ingest/interfaces.py

```diff
@@ -47,6 +47,8 @@ class BaseConnectorConfig(ABC):
     # where to write structured data outputs
     output_dir: str
     re_download: bool = False
+    metadata_include: str = ""
+    metadata_exclude: str = ""
```

Comment:
This will mean that the mutual exclusivity is only actually defined at the click() interface level. By testing `if metadata_exclude` first in the BaseIngestDoc, I think your logic still works, but I wonder if it'd be helpful to be more explicit here? Make these say Optional(str) with default None to distinguish between the user specifying an empty string versus not setting the value. You could then also validate exclusivity there by actually testing `is None`. Thoughts?

Reply:
Yeah, that's a really good idea to make them optional string and more explicitly signal the user that at most one of them should be set. Just pushed a fix.

Reply:
awesome! taking a look now.